package.json

```diff
@@ -90,8 +90,8 @@
     "@codeshift/test-utils": "^0.3.0",
     "@codeshift/utils": "^0.2.0",
     "@testing-library/jest-dom": "^5.7.0",
-    "@testing-library/react": "^10.0.4",
-    "@types/enzyme": "^3.10.5",
+    "@testing-library/react": "^10.4.9",
```

Comment:
nit: are we able to upgrade to `14.0.0` (or whatever newest version is without anything breaking? Wonder if we can get best of both worlds ;)

Reply:
Tried to install the latest version; however it throws an error due to React16 being in the dependent tree

```
npm install --save-dev @testing-library/react@14.1.2
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: react-resource-router@0.25.0
npm ERR! Found: react@16.14.0
npm ERR! node_modules/react
npm ERR!   dev react@"^16.10.2" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^18.0.0" from @testing-library/react@14.1.2
npm ERR! node_modules/@testing-library/react
npm ERR!   dev @testing-library/react@"14.1.2" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR!
npm ERR! For a full report see:
npm ERR! /Users/akaran/.npm/_logs/2023-11-22T02_25_00_700Z-eresolve-report.txt
```

Reply:
ah makes sense, 14 is dependent on 18